converter/internal/staticconvert/testdata/integrations_no_rw.yaml

```diff
@@ -0,0 +1,4 @@
+integrations:
+  node_exporter:
+    scrape_integration: true
```

Comment:
Interesting feature. If we don't support it, then this is technically a feature parity issue because customers might want to use it to control their DPM.

That said, I wouldn't want to support it in Flow in the same way Static mode does it. We need to make a new component which can expose metrics on an HTTP endpoint using the Prometheus exposition format.